root/release/edit/information.tt

```diff
@@ -51,33 +51,43 @@
         </td>
       </tr>
 
-      <tr data-bind="if: $root.loadingDuplicateRGs()">
+      <tr data-bind="if: $root.loadingDuplicateReleaseGroups()">
```

Comment:
I thought my earlier mix of "ReleaseGroup" and "RG" felt ugly, so I'm using "ReleaseGroup" everywhere now. Let me know if these feel too wordy, though.